index.js

```diff
@@ -3,10 +3,20 @@ const fetch = require('node-fetch');
 const { fromUrl } = require('hosted-git-info');
 const chalk = require('chalk');
 const { argv } = require('yargs');
+const fs = require('fs');
 
 const ISSUE_COUNT = 15;
 const MAX_CONCURRENT_REQUESTS = 10;
 
+function outputToFile(array) {
+  let file = fs.createWriteStream('./OUTPUT.html', { flags: 'w' });
+  file.on('error', function (err) {
+    console.log(err);
+  });
+  array.forEach((value) => file.write(`${value}\r\n`));
+  file.end();
+}
+
```

Comment:
Nice work using streams to write the file! I have an idea to make it even more flexible. We can leverage [I/O redirection](https://en.wikipedia.org/wiki/Redirection_(computing)) and let the operating system take care of writing the file for us. This has a couple of advantages:

* The user can choose the file name and file path of the desired output file.
* The user can use [unix pipes](https://en.wikipedia.org/wiki/Pipeline_(Unix)) or similar to pass our text output to other programs for further processing if desired.
* It makes our code simpler! :)

So here's what I would recommend:

We rename the `tofile` argument to `format`. If the value of the `format` argument is `html`, we simply output the HTML you've already generated below via `console.log`. We can then remove this function altogether.

Then, the command to generate the HTML file would look something like this:

```
npx shoulders --format html > ./my-file.html
```

(This will also allow us to expand the `format` argument in the future to support other formats, like `md`, `json`, `csv`, etc.)

Let me know what you think!

Reply:
Hey Matt!

Thanks for the very thoughtful reply, I appreciate it. I've redone #8 as per your suggestion and I feel a lot happier about it, I hope you will to.

I'll try to get these conflicts resolved and submit another PR ASAP!